mindjunkies/forums/views.py

```
@@ -0,0 +1,103 @@
+from django.contrib import messages
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from mindjunkies.courses.models import Course
+from .models import ForumTopic, ForumReply
+from .forms import ForumTopicForm, ForumReplyForm
+
+
+class ForumHomeView(LoginRequiredMixin, TemplateView):
+    template_name = "forums/forum_home.html"
+
+    def get_context_data(self, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_context_data`

(ANN201)